packages/framework-types/src/searcher.ts

```diff
@@ -33,13 +41,14 @@ export class Searcher<TObject> {
    * @param property The property the filter will act upon
    * @param operation The filter operation.
    * @param values The values for the filter. Depending on the operation, you can specify here one or many values
+   * @deprecated Use "filter" instead
```

Comment:
Why deprecating it if we're changing the name anyway producing breaking changes? Would it make sense to remove this code entirely? What would be the consequences? Is other part of the system using it? Is this to avoid making the PR too long? If that's the case, make sure you add an issue to remove this deprecated code in the near future!

Reply:
Because it's still used in the rest of the providers, so it's renamed in order to be able to make them know it's an old and deprecated functionality. I just created a new issue explaining that we should migrate all providers to the new system before removing it. https://github.com/boostercloud/booster/issues/601